README.md

```diff
@@ -1,9 +1,126 @@
+# kotsu: lightweight framework for structuring model validation
+
 [![PyPI version](https://img.shields.io/pypi/v/kotsu.svg)](https://pypi.org/project/kotsu/)
 ![lint-test status](https://github.com/datavaluepeople/kotsu/actions/workflows/run-ci.yml/badge.svg?branch=main)
 [![codecov](https://codecov.io/gh/datavaluepeople/kotsu/branch/main/graph/badge.svg?token=3W8T5OSRZZ)](https://codecov.io/gh/datavaluepeople/kotsu)
 
-# kotsu
-Lightweight framework for structured and repeatable model validation
+## What is it?
+
+**kotsu** is Python package that provides a lightweight and flexible framework to structure
+validating and comparing machine learning models. It aims to provide the skeleton on which to
+develop any kinds of models and to validate them in a robust and repeatable way, **without bloat or
+overheads**. Its flexibility fully leaves open usage with **any model interface**, and any
+complexity of validation, while the structure it provides **avoids common pitfalls** that occur
+when attempting to make fair comparisons between models.
+
+## Main Feature
+
+  - Register a model with hyperparameters to a unique ID
+  - Register validations to a unique ID
+  - Run all registered models through all registered validations, and have the results compiled and
+    stored into a dataframe
+  - Optionally passes an `artefacts_directory` to your validations, for storing of outputs for
+    further analysis, e.g. trained models, model predictions on test data sets
+  - Doesn't enforce any constraints or requirements on your models' interfaces
+  - Pure Python package, with no other setup or configuration of other systems required
+
+## Usage
+
+Import your packages for modelling and kotsu:
+
+```python
+from sklearn import datasets, svm
+from sklearn.model_selection import cross_val_score
+
+import kotsu
```

Comment:
🐼 people importing kotsu will be importing it as third party library so will be at the top of the third party section in this case

Reply:
60dee69